.travis.yml

```diff
@@ -1,7 +1,6 @@
 language: php
 
 php:
-  - "7.2"
```

Comment:
can you also reactivate these? I wasnt able to make a suggestion for them

Reply:
Fixed!